crates/polars-time/src/period_end.rs

```diff
@@ -0,0 +1,161 @@
+use arrow::legacy::time_zone::Tz;
+use chrono::NaiveDateTime;
+use polars_core::prelude::*;
+use polars_core::utils::arrow::temporal_conversions::{
+    timestamp_ms_to_datetime, MILLISECONDS, SECONDS_IN_DAY,
+};
+
+use crate::period_start::{
+    get_timestamp_closures, roll_month_backward, roll_quarter_backward, roll_year_backward,
+};
+use crate::windows::duration::Duration;
+
+// roll forward to the last day of the month
```

Comment:
can we either copy this comment to the respective functions below, or remove it?